.github/workflows/ci.yml

```diff
@@ -273,3 +273,9 @@ jobs:
           node-version: 16.x
       - run: yarn install --frozen-lockfile
       - run: npx semantic-release
+      - id: sequelize
+        uses: sdepold/github-action-get-latest-release@master
+        with:
+          repository: sequelize/sequelize
+      - run: |
+          curl -XPOST -u "sdepold:${{ secrets.GH_TOKEN }}" -H "Accept: application/vnd.github.everest-preview+json" -H "Content-Type: application/json" https://api.github.com/repos/sequelize/sequelize/dispatches --data '{"event_type":"Release notifier","client_payload":{"release-id": ${{ steps.sequelize.outputs.id }}}}'
```

Comment:
Why do you want the release ID? Isn't the tag enough?

Reply:
The notification script (https://github.com/sequelize/sequelize/blob/main/.github/workflows/notify.yml) will read the necessary meta information based on the release id.